src/api/app/assets/javascripts/webui/request.js

```diff
@@ -185,10 +185,9 @@ function loadDiffs(element){
   });
 }
 
-function loadChanges() { // jshint ignore:line
+function loadChanges(requestId, requestActionId) { // jshint ignore:line
   $('.loading-diff').removeClass('invisible');
-  var element = $('#changes-tab');
-  var url = '/request/' + element.data('request-number') + '/request_action/' + element.data('request-action-id') + '/changes';
+  var url = '/request/' + requestId + '/request_action/' + requestActionId + '/changes';
```

Comment:
@ncounter this route is using the bs_request.number, not the bs_request.id. Most of the time its the same, but I'm not sure if we can rely on it. So better use the number as intended for this route https://github.com/openSUSE/open-build-service/blob/809d4f0930b8048e730c0dc5ceef3e97e1eda7ec/src/api/config/routes/webui_routes.rb#L303

Reply:
A-ha!! That's a good catch. Thank you. Fixing..